.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.3.0
+  hooks:
+    - id: check-ast
+    - id: debug-statements
+    - id: end-of-file-fixer
+    - id: double-quote-string-fixer
+    - id: trailing-whitespace
+
+- repo: https://gitlab.com/pycqa/flake8
+  rev: 3.7.9
+  hooks:
+    - id: flake8
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.902
+  hooks:
+    - id: mypy
+      exclude: ^tests/
+      args:
+        [--no-strict-optional, --warn-unused-ignores, --ignore-missing-imports]
+
+- repo: https://github.com/pre-commit/mirrors-autopep8
+  rev: v1.7.0
+  hooks:
+    - id: autopep8
+
+- repo: https://github.com/asottile/add-trailing-comma
+  rev: v2.3.0
+  hooks:
+    - id: add-trailing-comma
+
+- repo: https://github.com/asottile/pyupgrade
+  rev: v3.1.0
+  hooks:
+    - id: pyupgrade
+
+- repo: https://github.com/asottile/reorder_python_imports
+  rev: v3.8.4
+  hooks:
+    - id: reorder-python-imports
```

Comment:
Please take a look if there are args (options) that could be useful (in view of the fact that this repository aims to support Python version >= 3.7)